content/en/docs/tasks/administer-cluster/enabling-endpointslices.md

```diff
@@ -39,36 +39,29 @@ an outright replacement.
 {{< /note >}}
 
 EndpointSlice functionality in Kubernetes is made up of several different
-components, most are enabled by default:
+components:
 * _The EndpointSlice API_: EndpointSlices are part of the
-  `discovery.k8s.io/v1beta1` API. This is beta and enabled by default since
-  Kubernetes 1.17. All components listed below are dependent on this API being
-  enabled.
+  `discovery.k8s.io/v1` API. This is stable starting from Kubernetes 1.21.
+  All components listed below are dependent on this API.
 * _The EndpointSlice Controller_: This {{< glossary_tooltip text="controller"
   term_id="controller" >}} maintains EndpointSlices for Services and the Pods
-  they reference. This is controlled by the `EndpointSlice` feature gate. It has
-  been enabled by default since Kubernetes 1.18.
+  they reference.
 * _The EndpointSliceMirroring Controller_: This {{< glossary_tooltip
   text="controller" term_id="controller" >}} mirrors custom Endpoints to
-  EndpointSlices. This is controlled by the `EndpointSlice` feature gate. It has
-  been enabled by default since Kubernetes 1.19.
+  EndpointSlices.
 * _Kube-Proxy_: When {{< glossary_tooltip text="kube-proxy" term_id="kube-proxy">}}
   is configured to use EndpointSlices, it can support higher numbers of Service
   endpoints. This is controlled by the `EndpointSliceProxying` feature gate on
```

Comment:
Make sure that _content/en/docs/reference/command-line-tools-reference/feature-gates.md_ lists `EndpointSliceProxying`.

Reply:
`EndpointSliceProxying` is in the alpha, beta feature gate list.